core/arch/arm/plat-imx/conf.mk

```diff
@@ -0,0 +1,36 @@
+include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
+
+CROSS_COMPILE	?= arm-poky-linux-gnueabi-
+COMPILER	?= gcc
+
+ifeq ($(CFG_ARM64_core),y)
+CFG_WITH_LPAE := y
+else
+CFG_ARM32_core ?= y
+CFG_MMU_V7_TTB ?= y
+endif
+
+core-platform-cppflags	 = -I$(arch-dir)/include
+core-platform-subdirs += \
+	$(addprefix $(arch-dir)/, kernel mm tee sta) $(platform-dir)
+core-platform-subdirs += $(arch-dir)/sm
+
+libutil_with_isoc := y
+CFG_SECURE_TIME_SOURCE_CNTPCT := y
+CFG_WITH_STACK_CANARIES := y
+CFG_NO_TA_HASH_SIGN ?= y
+CFG_IMX_UART ?= y
+CFG_MMU_V7_TTB ?= y
+CFG_GENERIC_BOOT ?= y
+CFG_PM_STUBS ?= y
+
+include mk/config.mk
+include $(platform-dir)/system_config.mk
+
+CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
+
+core-platform-cppflags += -DTEE_USE_DLMALLOC
```

Comment:
Will remove the 'core-platform-cppflags += -DTEE_USE_DLMALLOC'
I grep CFG_TEE_CORE_EMBED_INTERNAL_TESTS, and get the following outputs, will set it to n.
```
./core/arch/arm/plat-hikey/conf.mk:24:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= n
./core/arch/arm/plat-vexpress/conf.mk:29:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
./core/arch/arm/plat-sunxi/conf.mk:20:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
./core/arch/arm/sta/sub.mk:1:srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += sta_self_tests.c
./core/arch/arm/sta/sub.mk:2:srcs-$(CFG_TEE_CORE_EMBED_INTERNAL_TESTS) += core_self_tests.c
./core/arch/arm/plat-stm/conf.mk:20:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= y
./core/arch/arm/plat-mediatek/conf.mk:24:CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= n
```